webapp/app/assessment/assessmentState.ts

```diff
@@ -52,7 +52,7 @@ export const isLocked = (assessment: any) => (state: any) => {
 }
 
 export const canToggleLock = (assessment: any) => (state: any) => {
```

Comment:
better to refactor this to hook

Reply:
should they be done in this pr or in assessment refactor?